packages/datagateway-dataview/src/views/table/isis/isisStudiesTable.component.tsx

```diff
@@ -0,0 +1,217 @@
+import FingerprintIcon from '@material-ui/icons/Fingerprint';
+import DescriptionIcon from '@material-ui/icons/Description';
+import CalendarTodayIcon from '@material-ui/icons/CalendarToday';
+
+interface ISISStudiesTableProps {
+  instrumentId: string;
+}
+
+interface ISISStudiesTableStoreProps {
+  sort: SortType;
+  filters: FiltersType;
+  data: Entity[];
+  totalDataCount: number;
+  loading: boolean;
+  error: string | null;
+  allIds: number[];
+}
+
+interface ISISStudiesTableDispatchProps {
+  pushSort: (sort: string, order: Order | null) => Promise<void>;
+  pushFilters: (filter: string, data: Filter | null) => Promise<void>;
+  fetchData: (allIds: number[], offsetParams: IndexRange) => Promise<void>;
+  fetchCount: (allIds: number[]) => Promise<void>;
+  fetchIds: (instrumentId: number) => Promise<void>;
+}
+
+type ISISStudiesTableCombinedProps = ISISStudiesTableProps &
+  ISISStudiesTableStoreProps &
+  ISISStudiesTableDispatchProps;
+
+const ISISStudiesTable = (
+  props: ISISStudiesTableCombinedProps
+): React.ReactElement => {
+  const {
+    allIds,
+    data,
+    totalDataCount,
+    fetchIds,
+    fetchData,
+    fetchCount,
+    sort,
+    pushSort,
+    filters,
+    pushFilters,
+    instrumentId,
+    loading,
+  } = props;
+
+  const [t] = useTranslation();
+
+  const textFilter = (label: string, dataKey: string): React.ReactElement => (
+    <TextColumnFilter
+      label={label}
+      value={filters[dataKey] as string}
+      onChange={(value: string) => pushFilters(dataKey, value ? value : null)}
+    />
+  );
+
+  const dateFilter = (label: string, dataKey: string): React.ReactElement => (
+    <DateColumnFilter
+      label={label}
+      value={filters[dataKey] as DateFilter}
+      onChange={(value: { startDate?: string; endDate?: string } | null) =>
+        pushFilters(dataKey, value ? value : null)
+      }
+    />
+  );
+
+  React.useEffect(() => {
+    fetchIds(parseInt(instrumentId));
+  }, [fetchIds, instrumentId]);
+
+  React.useEffect(() => {
+    fetchCount(allIds);
+    fetchData(allIds, { startIndex: 0, stopIndex: 49 });
+  }, [fetchCount, fetchData, sort, filters, allIds]);
+
+  const pathRoot = 'browseStudyHierarchy';
+  const instrumentChild = 'study';
+
+  return (
+    <Table
+      loading={loading}
+      data={data}
+      loadMoreRows={(params) => fetchData(allIds, params)}
+      totalRowCount={totalDataCount}
+      sort={sort}
+      onSort={pushSort}
+      columns={[
+        {
+          icon: <TitleIcon />,
+          label: t('studies.name'),
+          dataKey: 'STUDY.NAME',
+          cellContentRenderer: (props: TableCellProps) =>
+            tableLink(
+              `/${pathRoot}/instrument/${instrumentId}/${instrumentChild}/${props.rowData.ID}/investigation`,
+              props.rowData.STUDY?.NAME
+            ),
+          filterComponent: textFilter,
+        },
+        {
+          icon: <FingerprintIcon />,
+          label: t('studies.rb_number'),
```

Comment:
Again, the study RB number is the name. Studies don't have visit ids, since one study could contain multiple visits